core.yaml

```diff
@@ -239,17 +375,25 @@ adminApps:
         port: 8080
         type: public
         auth: true
-  - name: loki
+    info:
+      appVersion: 2.3.0
+      repo: https://github.com/openclarity/kubeclarity
+      maintainers: OpenClarity
+      relatedLinks: [https:otomi.io]
+      license: Apache 2.0
+      about: KubeClarity is a tool for detection and management of Software Bill Of Materials (SBOM) and vulnerabilities of container images and filesystems. It scans both runtime K8s clusters and CI/CD pipelines for enhanced software supply chain security.
+      integration: KubeClarity can be activated to scan running containers and filesystems for vulnerabilities.
+  - name: grafana
```

Comment:
You can't just change the name of an app here for multiple reasons:
- you break the code
- you overwrite it later with another `grafana:` section.
- 
```suggestion
  - name: loki
```